grouper/fe/templates/macros/ui.html

```diff
@@ -152,9 +153,9 @@ <h3 class="panel-title">Groups</h3>
     <tr>
         <td>
             {% if member.name != current_user.name and
-                (current_user.my_role(members) == "owner" or
+                (current_user.my_role(members) in ("owner", "np-owner") or
                  (current_user.my_role(members) == "manager" and
-                  ROLES[member.role] == "member"))
+                  ROLES[member.role] in ("member", "manager")))
```

Comment:
As mentioned, the idea of managers was that they can only control members. The management structure is controlled by the owners. (This can obviously change, but I don't want it changed because someone thought it was a bug -- it's a deliberate decision.)